handlers/upload.go

```diff
@@ -22,11 +22,13 @@ import (
 )
 
 type UploadVODRequestOutputLocationOutputs struct {
-	SourceMp4          bool `json:"source_mp4"`
-	SourceSegments     bool `json:"source_segments"`
-	TranscodedSegments bool `json:"transcoded_segments"`
-	ForceMP4           bool `json:"force_mp4"`
-	AutoMP4            bool `json:"auto_mp4"`
+	SourceMp4          bool   `json:"source_mp4"`
+	SourceSegments     bool   `json:"source_segments"`
+	TranscodedSegments bool   `json:"transcoded_segments"`
+	ForceMP4           bool   `json:"force_mp4"`
+	AutoMP4            bool   `json:"auto_mp4"`
+	HLS                string `json:"hls"`
+	MP4                string `json:"mp4"`
```

Comment:
I can't quite tell what the default behaviour is if both of these are empty. Or is an error returned?